go.mod

```diff
@@ -292,6 +292,8 @@ require (
 
 replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	// dgrijalva/jwt-go is deprecated and doesn't receive security updates.
```

Comment:
what is this used by? we should identify who is using this and get them to update instead of adding more and more replace tags

Reply:
why not just use `github.com/golang-jwt/jwt/v4` directly?

Reply:
we don't use it, it's from a dependency.

Reply:
it will take more time to go through dependencies update. But yes - ideally we figure out which dependency is using it and maybe check if there is a new version.

Reply:
`go mod graph` yields the following:

```
go.etcd.io/etcd@v0.0.0-20191023171146-3cf2f69b5738 github.com/dgrijalva/jwt-go@v3.2.0+incompatible
github.com/go-kit/kit@v0.10.0 github.com/dgrijalva/jwt-go@v3.2.0+incompatible
```

Reply:
ah perfect this will be removed with cosmos-db, we can merge then need to remember to remove later on

Reply:
go-kit and etcd is brought by cosmos-db?